backend/src/middleWare/auth.js

```diff
@@ -17,7 +18,7 @@ const auth = async (req, res, next) => {
         next()
 
     } catch (e) {
-        res.status(401).send({ error: "Please authenticate" })
+        res.status(401).send(ERRORS.UNAUTHORIZED.toString())
```

Comment:
why the to String ?

Reply:
because when i leave it as a number it doesn't show up on the screen when i receive the error

Reply:
try to find a way to make it work, if you cant then i would say change the Errors object to have "300" instead of 300, so we can avoid the toString() everytime